### PR TITLE
feat(task-orchestrator): TO-CANON 005 coldstart materialization + 007 cutover plan

### PR DESCRIPTION
**Pull request by hu3mann**
## Summary

Continues the **TO-CANON** series (TP-TO-CANON-000→004 + 004R already MERGED via #961). This PR lands the two **offline / docs-only** tail packets:

- **TP-TO-CANON-005** — deterministic, offline Markdown renderer over the committed `COLDSTART_RECONCILIATION.json` + the generated decision report. No raw pack, no live `current-tasks.db`.
- **TP-TO-CANON-007** — plan-only, operator-gated canonical-store **cutover plan** + red-team risk doc (verdict PASS_WITH_RISKS).
- Adds `docs/ops/load-plans/load_plan-TO-CANON.json` documenting the full 000→007 DAG.

`TP-TO-CANON-006` (read-only operator view behind a feature flag) ships separately as runtime code.

## Validation (PASS)
- `python -m jsonschema -i task-packets/TP-TO-CANON-00{5,7}.json docs/03-reference/spec/dopetask/dopetask-canonical-spec.json` — both valid
- `pytest tests/tools/test_task_orchestrator_reconcile_coldstart.py tests/tools/test_task_orchestrator_reconcile_import.py` — **13 passed** (6 new renderer + 7 existing importer regression)
- Byte-stability: re-rendering the report from the committed JSON is byte-identical
- `scripts/docs_validator.py` on all new docs — exit 0
- `pre-commit run` on staged files — all applicable hooks pass
- `git diff --check` — clean

## Audit note
The 005 renderer was implemented by a Sonnet subagent and audited by Opus: an **unreachable-code defect** (dead `return 2` after `parser.exit`, which Pyright flagged) was fixed to use `parser.error()` (`NoReturn`) before commit; verification re-run from scratch.

## Out of scope (by design)
- No live `current-tasks.db` writes; no fix of the 100/101 status-label drift; no unblock of `INIT-UNIFY-102` (its packet was never landed on main).
- No write cutover — 007 is a plan only.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by hu3mann** [comment]
## Deep audit finding: Markdown renderer should escape table cells

Current head: `8b376ffabd76`.

OBSERVED: `tools/task_orchestrator_reconcile/coldstart.py` renders item rows directly into a Markdown table at the current head. `title`, `role`, `decision`, `status_label`, and the evidence string are interpolated without escaping `|`, newlines, or carriage returns.

Current committed output happens to render cleanly, but the renderer is also wired as a reusable `--emit-coldstart-md` path from `import_pack.py`. Future Task Orchestrator titles or evidence values containing table delimiters would silently corrupt the report table while still producing a byte-stable file.

IMPACT: this is not a live-state mutation risk, but it is an auditability/replayability risk for a generated proof/report surface. A malformed table can misrepresent classification rows or evidence cells without failing validation.

REQUESTED FIX: add a small Markdown table-cell escape helper and use it for every interpolated cell in `render_markdown()`. Add a regression test with a title/evidence value containing `|` and a newline so the table shape is preserved deterministically.

**Comment by hu3mann** [comment]
Remediation update for the audit blocker above:

- Added deterministic Markdown table-cell escaping for all interpolated item cells in `render_markdown()`: title, role, decision, status label, and evidence.
- Added regression coverage with `|` and newline content to verify the item row keeps its table shape.
- Commit: `7b91dcb3a03832b29ee3170f321a603d1c73deef`.

Validation run locally:

```text
python -m pytest -q tests/tools/test_task_orchestrator_reconcile_coldstart.py
# 7 passed

git diff --check
# passed before commit

pre-commit run --files tools/task_orchestrator_reconcile/coldstart.py tests/tools/test_task_orchestrator_reconcile_coldstart.py
# passed
```

**Comment by hu3mann** [comment]
Codex review items + table-escaping audit finding are resolved:

- `7b91dcb3a` — Markdown table cells escape pipes/newlines via `_markdown_table_cell`; regression test covers title/role/decision/status/evidence.
- `9651a50cd` — Phase 1 backup uses SQLite `.backup` (not bare cp).
- `33151efb7` — TP-TO-CANON-007 `depends_on` is `["TP-TO-CANON-006"]`.
- `f6d2bc6ff` — Rollback uses `.restore`; R8 forbids bare cp-a.

Classification: AUTO_APPLIED verified at current head.

File: task-packets/TP-TO-CANON-007.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "Docs-only. No code, no live current-tasks.db mutation, no actual cutover.",
+    "The plan must require explicit operator approval before any write/cutover.",
+    "Stale/recovery DBs are archived, never deleted.",
+    "Authority boundaries (task-orchestrator views-only, Leantime passive PM, ConPort structured context/decisions, dope-memory chronicle, dopecon-bridge transport-only) are preserved and named.",
+    "dopecon-bridge must not be promoted into a data authority.",
+    "The plan names the canonical writer, a live backup-before-mutation step, and exact rollback commands.",
+    "Enabling and validating the TP-TO-CANON-006 read-only view is listed as a cutover precondition."
+  ],
+  "depends_on": [
+    "TP-TO-CANON-004"
```
Comment (hu3mann) [inline task-packets/TP-TO-CANON-007.json]: [medium] `depends_on` lists only `TP-TO-CANON-004`, but this packet's own invariant ("Enabling and validating the TP-TO-CANON-006 read-only view is listed as a cutover precondition") and the new `load_plan-TO-CANON.json` DAG both require `006 → 007`. As written, if 007 is loaded/scheduled from its own `depends_on`, it unblocks after 004 and can run before the 006 read-view precondition it is supposed to depend on. Add `"TP-TO-CANON-006"` to `depends_on` so the packet matches the load-plan DAG and the stated invariant. (Offline planning artifact — not a merge blocker, but it is real contract drift inside this PR.)

**Review by hu3mann — COMMENTED** [review]
> 🤖 **Automated thorough review** — Claude Code (Opus 4.8). Verdict: **comment** · Blocking issues: **0**. Inline comments anchor the key findings; full analysis + cross-PR conflicts below.

## Overview
TP-TO-CANON-005 adds a deterministic, offline `render_markdown(coldstart: dict) -> str` plus a `--from-json/--emit-md` CLI and an `--emit-coldstart-md` hook in `import_pack.py`; it materializes the committed `COLDSTART_RECONCILIATION.json` into a tracked decision report. TP-TO-CANON-007 is plan-only docs (cutover plan + risk doc, verdict PASS_WITH_RISKS). Plus `load_plan-TO-CANON.json` and the two packets. All 9 files are additive (+970/-0).

The executable surface is correct, deterministic, offline, and tested. The non-blocking findings are all in plan-only docs or weak verify recipes.

## Verified strengths (primary-source)
- **Determinism/replay holds**: extracted the head-commit renderer + the head-commit fixture and re-rendered — output is **byte-identical** to the committed `coldstart-reconciliation-20260622.md` (4800 bytes, diff clean). Materialization is reproducible.
- **Fixture is tracked** in git at the head tree and on `main` (`audit_inputs/.../to-all-dbs-20260622T192814Z/COLDSTART_RECONCILIATION.json`) — CI tests have their input even though it is absent from this shared dirty worktree.
- **Escaping fix is correct** (commit 7b91dcb3a): executing `_markdown_table_cell`/`_evidence_cell` — content `|` → `\|`, `\n`/`\r\n`/`\r` normalized to `<br>`, exactly 6 unescaped column delimiters per 5-column row. Regression test asserts this.
- **Offline/read-only by construction**: renderer takes a dict; no `sqlite3.connect` on a live DB in the new paths.
- **007 fails closed**: explicit operator-approval gate (default denied), Phase 3 write cutover marked NOT AUTHORIZED, authority-boundary table normative, canonical writer named (upstream TO MCP, `authority_status: CONFLICTING`), bridge kept transport-only.
- **All cross-links + referenced config resolve** at head; docs_validator/Documentation Check pass.

## Findings
- **[medium] `task-packets/TP-TO-CANON-007.json:15`** — `depends_on` lists only `TP-TO-CANON-004`, but the packet's own invariant ("Enabling and validating the TP-TO-CANON-006 read-only view is listed as a cutover precondition") and the load-plan DAG (`006 → 007`) require 006 first. Internal contract drift inside this PR: if 007 is ever loaded/scheduled from its own `depends_on`, it unblocks after 004 and can run before its stated precondition. Add `TP-TO-CANON-006` to `depends_on`. (Offline planning artifact, not executed → medium, not blocker.)
- **[medium] `docs/03-reference/systems/task-orchestrator/canonical-store-cutover-plan.md:86-87`** — Phase-1 backup uses `cp -a current-tasks.db` then `PRAGMA integrity_check`. For WAL-mode SQLite this is not a full backup: `-wal`/`-shm` sidecars are not copied and recent committed transitions can still live in `current-tasks.db-wal`, so integrity_check passes on a stale main file and a post-cutover rollback restores stale state. The repo wrapper itself flags WAL-lock contention on this DB. Use `.backup`/online backup or checkpoint+quiesce and copy sidecars. Plan-only/NOT AUTHORIZED, but documents an incorrect operator command.
- **[low] cutover-plan.md:84** — `ARCHIVE="audit_inputs/.../backups/$(git rev-parse --short HEAD)"` is described as a "timestamped archive directory" but is keyed only on the short SHA; re-running Phase 1 from the same commit overwrites the prior backup, losing the earlier rollback point. Add a UTC timestamp/run-id.
- **[low] `task-packets/TP-TO-CANON-005.json` verify block** — the byte-stability recipe writes generated Markdown over the tracked file (`--emit-md docs/.../coldstart-...md`) and then diffs that overwritten file against a second fresh render to `/tmp`. It never compares to HEAD, so a stale committed report or unintended renderer change would still pass. Render to temp + diff against the checked-in file before overwriting, or add a clean-tree git check.
- **[nit] `tools/task_orchestrator_reconcile/coldstart.py:887-899`** — `render_markdown` hardcodes frontmatter (`id`, `date: '2026-06-22'`, etc.). Correct for this point-in-time report, but `import_pack.py --emit-coldstart-md` reuses the same function, so any future import of a different pack would emit the fixed 2026-06-22 frontmatter. Latent footgun for the reusable path; consider deriving id/date from the pack's `point_in_time`.

## Test coverage
6 new renderer tests + the escaping regression test in `tests/tools/test_task_orchestrator_reconcile_coldstart.py`: determinism (in-process + CLI byte-identical), classification correctness (100/101/109 = accepted_do_not_rerun, 102 = keep_blocked, operator gate present), `valid_as_of` banner verbatim, and delimiter/newline escaping. Adequate for the offline renderer. Gaps (non-blocking): no test for the `import_pack.py --emit-coldstart-md` wiring; no malformed/missing-`point_in_time` failure-path test (KeyError would propagate); empty-group rendering (`_No items_`) untested. 007/load-plan are docs — covered by docs_validator + jsonschema, not pytest.

## Validation
- **PASS** — byte-identical regeneration of committed report from committed fixture via head-commit renderer (verified directly).
- **PASS** — escaping logic produces 6 unescaped delimiters, escapes content `|`→`\|`, newlines→`<br>` (executed directly).
- **PASS** — fixture tracked in git head tree + main; cross-links and `config/runtime_authority_manifest.json` resolve.
- **PASS** — CI: 24 checks green, 3 skipped, 0 failing.
- **NOT_RUN** — full pytest suite in this worktree (fixture absent from dirty shared checkout; CI ran it green). The 13-passed claim in the PR body is consistent with CI's Unit Tests pass.

## Verdict
**comment** — approve-with-non-blocking-nits. The executable materialization is deterministic, offline, replay-safe, and well-tested; 007 is a correctly fail-closed plan-only doc. No blockers. The medium findings (007 `depends_on` should include 006; WAL-unsafe `cp` backup) are worth fixing before this becomes the basis for any real cutover, but none gate the merge of these offline/docs artifacts.

---

### 🔗 Cross-PR analysis

**#971** ("TO-CANON 006 read-only operator view") is the runtime-code sibling. File overlap with #970: **NONE** — #971 touches `src/dopemux/commands/orchestrator_commands.py`, `src/dopemux/orchestrator/canonical_readview.py`, `task-packets/TP-TO-CANON-006.json`, `tests/unit/orchestrator/test_canonical_store_readview.py`; #970 touches packets 005/007, `coldstart.py`, `import_pack.py`, the report, the cutover/risk docs, and `load_plan-TO-CANON.json`. No merge conflict, no shared file. **Logical coupling:** #970's cutover plan (007) and load-plan DAG reference TP-006's `CANONICAL_STORE_READ_VIEW_ENABLED` flag/surface as a precondition, but that flag/code only lands in #971. If #970 merges first, the cutover plan forward-references a flag not yet in the codebase — acceptable because 007 is explicitly NOT AUTHORIZED / plan-only, but it argues for merging #971 (006) before or together with #970 to keep the docs self-consistent. The 005 renderer/report does NOT read anything #971 produces (005 consumes the committed offline JSON, not the 006 read-view) — no data dependency. **Recommended order:** #971 (006) then #970 (007 plan), or land together. (Series body confirms 000–004+004R already MERGED via #961; 005/006/007 are the tail.) Not coupled to 933/943/949/957/958/966/967/968.

File: task-packets/TP-TO-CANON-007.json (hunk 1 above)
Comment (hu3mann) [reply task-packets/TP-TO-CANON-007.json]: Addressed at 33151efb and retained: TP-TO-CANON-007 depends_on is ["TP-TO-CANON-006"] matching the cutover precondition invariant. Thread was outdated; resolving.

File: docs/03-reference/systems/task-orchestrator/canonical-store-cutover-plan.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+```bash
+# 1. Disable the read view (default is already off):
+unset CANONICAL_STORE_READ_VIEW_ENABLED
+
+# 2. Revert any code change introduced by a cutover step:
+git revert <cutover-commit-sha>
+
+# 3. Restore a live DB from backup ONLY under explicit operator direction (only if a write occurred):
+cp -a "$ARCHIVE/current-tasks.<workspace-id>.db" "<live current-tasks.db path>"
```
[the remark this answers is not in the document]
Comment (hu3mann) [reply docs/03-reference/systems/task-orchestrator/canonical-store-cutover-plan.md]: Addressed in f6d2bc6: rollback step 3 now uses sqlite3 .restore + integrity_check + WAL/SHM cleanup; aligned with Phase 1 no-bare-cp invariant.

File: docs/05-audit-reports/task-orchestrator/canonical-store-cutover-risk.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| R2 | Authority flattening — treating the derived store as the single PM truth | Low | High (breaks split-authority model) | Authority-boundary table is normative; store is DERIVED/read-only; canonical writer stays the live TO MCP |
+| R3 | `dopecon-bridge` promoted to a data authority | Low | High | Explicit invariant: bridge is transport-only and MUST NOT become an authority |
+| R4 | Replay / idempotency regressions after a write cutover | None now | Medium-High | Precondition 4 requires replay/dedupe verification before any write |
+| R5 | Stale / recovery DB rows promoted into active state | Low | Medium | Stale/legacy/recovery DBs stay provenance-only; archive-never-delete |
+| R6 | Operator-approval gate bypass / automation | Low | High | Plan fails closed; default denied; no step past Phase 2 without recorded sign-off |
+| R7 | Stale point-in-time view presented as live | Medium | Medium | Read view (006) carries a `valid_as_of` banner; cutover requires a freshly regenerated snapshot |
+| R8 | Task Orchestrator MCP health / SQLite contention during backup | Medium | Low-Medium | Backup uses read-only `cp -a` copies of the DB files, not live MCP calls |
```
[the remark this answers is not in the document]
Comment (hu3mann) [reply docs/05-audit-reports/task-orchestrator/canonical-store-cutover-risk.md]: Addressed in f6d2bc6: R8 mitigation now documents .backup/.restore and forbids bare cp -a for WAL-backed live DBs.
